components/lcd/esp_lcd_gc9503/README.md

```diff
@@ -19,63 +19,84 @@ You can add them to your project via `idf.py add-dependancy`, e.g.
 
 Alternatively, you can create `idf_component.yml`. More is in [Espressif's documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-component-manager.html).
 
-## Example use
+## Initialization Code
 
-Create 3-wire SPI panel IO using [esp_lcd_panel_io_additions](https://components.espressif.com/components/espressif/esp_lcd_panel_io_additions) component.
```

Comment:
Could we add a short paragraph that explains 3wire SPI + RGB interface and that we support GPIOs through IO expander?

Reply:
Sure, I added it. PTAL!